docs/03-tutorials/00-api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,302 @@
+   kind: DNSProvider
+   metadata:
+     name: dns-provider
+     namespace: $NAMESPACE
+   spec:
+     type: $SPEC_TYPE
+     secretRef:
+       name: $SECRET
+     domains:
+       include:
+         - $DOMAIN
+   EOF  
+   ```
+
+   </details>
+   </div>
+
+- Create a DNSEntry CR. Export values of the **metadata.namespace**, **metadata.spec.dnsName**, and **metadata.spec.targets.IP** parameters as environment variables. Optionally, you can also change the value of **metadata.spec.ttl**. Run:
+
+   ```bash
+   export NAMESPACE={NAMESPACE_NAME}
+   export SUBDOMAIN={YOUR_SUBDOMAIN}
+   export IP={SECRET_NAME}
+   ```
+
+   Create a DNSEntry CR. Run:
+
+  <div tabs>
+  <details>
+  <summary>
+  Gardener cluster
+  </summary>
+
+  ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSEntry
+   metadata:
+     name: dns-entry
+     namespace: $NAMESPACE
+     annotations:
+       dns.gardener.cloud/class: garden
+   spec:
+     dnsName: "$SUBDOMAIN"
+     ttl: 600
+     targets:
+       - $IP
+   ```
+
+  </details>
+  <details>
+  <summary>
+  Non-Gardener cluster
+  </summary>
+
+  ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSEntry
+   metadata:
+     name: dns-entry
+     namespace: $NAMESPACE
+   spec:
+     dnsName: "$SUBDOMAIN"
+     ttl: 600
+     targets:
+       - $IP
+   ```
+
+   </details>
+   </div>
+
+    >**NOTE:** To check the Ingress Gateway IP, run:
+    >```bash
+    >kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+    >```
+
+   >**NOTE:** You can create many DNSEntry CRs for one DNSProvider, depending on the number of subdomains you want to use.
+
+5. Create an Issuer CR. Export values of the **metadata.spec.acme.email**, **metadata.spec.domains.include**, and **metadata.spec.domains.exclude** parameters as environment variables. As the values for the **metadata.spec.domains.include** parameter, use the main domain, the subdomain from the **spec.dnsName** parameter of the DNSEntry CR, and a wildcard DNS record. Run:
+
+   ```bash
+   export EMAIL={YOUR_EMAIL_ADDRESS}
+   export DOMAIN={CLUSTER_DOMAIN}
+   export SUBDOMAIN={YOUR_SUBDOMAIN}
+   export WILDCARD={YOUR_WILDCARD_DNS_RECORD, e.g. *.mydomain.com}
+   ```
+
+   Create an Issuer CR. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: cert.gardener.cloud/v1alpha1
+   kind: Issuer
+   metadata:
+     name: letsencrypt-staging
+     namespace: default
+   spec:
+     acme:
+       server: https://acme-staging-v02.api.letsencrypt.org/directory
+       email: $EMAIL
+       autoRegistration: true
+       # Name of the Secret used to store the ACME account private key
+       # If doesn't exist, a new one is created
+       privateKeySecretRef:
+         name: letsencrypt-staging-secret
+         namespace: default
+       domains:
+         include:
+           - $DOMAIN
```

Comment:
I have doubts if we should include $DOMAIN here. Is it strictly necessary? I think cert issued for $SUBDOMAIN and $WILDCARD is enough. I will check it.